src/mono/msbuild/apple/build/AppleBuild.targets

```diff
@@ -6,9 +6,12 @@
              AssemblyFile="$(MonoTargetsTasksAssemblyPath)" />
 
   <PropertyGroup>
```

Comment:
Looks like this file will be part of library builder, there is still a bunch of refereneces to App in this file, since its more generic and shared, maybe we should make sure properties are aligned as much as possible to be more generic and not app centric?

Reply:
Modified to be more generic.